main/test/flix/Test.Exp.Effect.flix

```diff
@@ -100,6 +100,28 @@ namespace Test/Exp/Effect {
         }
     }
 
+    pub def singleEffect(): Bool \ Empty = ???
```

Comment:
Can we not call the effect empty, but instead `Flip` or whatever. I find it confusing :)